src/components/ComparisonTool/ComparisonModal.vue

```diff
@@ -90,94 +94,96 @@
                   md="12"
                   sm="7"
                 >
-                  <v-radio-group
-                    v-model="radios"
-                    mandatory
-                    dense
-                  >
-                    <div class="d-flex flex-wrap flex-xl-column flex-lg-column flex-md-column flex-sm-row flex-row">
-                      <v-radio
-                        v-for="(option, i) in options"
-                        :key="i"
-                        :label="option.label"
-                        :value="option.value"
-                        data-cy="radio-buttons"
-                        class="pr-6 mb-1"
-                      />
-                    </div>
-                    <v-menu
-                      v-model="menu1"
-                      :close-on-content-click="false"
-                      transition="scale-transition"
-                      offset-y
-                      min-width="auto"
-                      :nudge-left="260"
+                  <div class="demo_purpose">
```

Comment:
```suggestion
                  <div style="display: None;">
```
Do it this way so you don't need the <style> at the bottom of your vue file